Tests/SonarQube.Bootstrapper.Tests/ArgumentProcessorTests.cs

```diff
@@ -37,7 +37,49 @@ public void ArgProc_UnrecognizedArgumentsAreIgnored()
 
             // 1. Minimal command line settings with extra values
             IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "foo", "blah", "/xxxx");
-            AssertExpectedUrl("foo", settings);
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\" \"foo\" \"blah\" \"/xxxx\"");
+        }
+
+        [TestMethod]
+        public void ArgProc_InstallTargets()
+        {
+            TestLogger logger = new TestLogger();
+
+            //  No argument passed -> install targets
+            IBootstrapperSettings settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo");
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\"");
+
+            // "true"-> install targets
+            settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "/i:true");
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\"");
+
+            // Case insensitive "TrUe"-> install targets
+            settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "/i:TrUe");
+            AssertCommonSettings(settings, "foo", true, "\"/d:sonar.host.url=foo\"");
+
+            // "false"-> don't install targets
+            settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "/i:false");
+            AssertCommonSettings(settings, "foo", false, "\"/d:sonar.host.url=foo\"");
+
+            // Case insensitive "falSE" 
+            settings = CheckProcessingSucceeds(logger, "/d:sonar.host.url=foo", "/i:falSE");
+            AssertCommonSettings(settings, "foo", false, "\"/d:sonar.host.url=foo\"");
+
+            // Invalid value -> parsing should fail
+            logger = CheckProcessingFails("/d:sonar.host.url=foo", "/i:1");
+            logger.AssertErrorsLogged(1);
```

Comment:
You're expecting the error message to contain the /i: prefix, so you could use AssertSingleErrorExists to check for that.
The AssertSingleXXXExists() methods check that an error/warning/message exists that contains all of the supplied string fragments.
It's not a complete a check that the error message is exactly as expected, but it's useful sanity check that does catch cases where the parsing is failing for an unexpected reason.